### ISSUE TITLE
Bad Training Result

### ISSUE THREAD

**Issue by rohitpaul24** [jdh-algo/JoyVASA#34]
Hey, I was training a new model using your training script. Without any modification on my own dataset. 
Using the default parameters used in the training script
But after training for 50k steps, eventhough  there is a convergence in both training and val loss. The result has too much flikering(even on training set)

Here is an image to show the motion data index 0 (exp first value) value changes each frame
![Image](https://github.com/user-attachments/assets/4ec78298-16eb-4ca7-8011-e7449f650642)

Here is the training loss:

![Image](https://github.com/user-attachments/assets/faad9c6d-2499-4948-85d1-1d6ae95518a9)

It seems the loss is still high even after 50k steps. And potential too converge more.
Whether more training is required? Or there is an issue with the data
